other-docs/guides/getting-help-with-altis.md

```diff
@@ -1,50 +1,91 @@
 # Getting help with Altis
 
-If you need help, this guide will show you how to most effectively get you the help you need.
+Everybody needs a little help sometimes, and in this guide we'll inform you on how to open support requests from the Altis Support Team. 
 
 ## Altis Dashboard
 
-While we support a few methods, the primary method and the route you should choose, is the Altis Dashboard Support button.
+Support happens via the Altis Dashboard. From your Altis Dashboard you will be able create, view, and reply to open support requests. 
 
-<img width="1398" alt="Altis Dashboard support link" src="https://user-images.githubusercontent.com/1263370/113014595-a119f980-9174-11eb-95d6-0e2a9718c5e5.png">
+To open a support request, first you'll have to [login to the Altis Dashboard](dashboard.altis-dxp.com/).
 
-We break down support into different types. Each has their own language and processes, but they're somewhat intuitive once you internalize the language a little. Let's go over the types.
+From there, you'll find the link to the support page in the top right corner. See below.
 
-<img width="1282" alt="Altis Dashboard Create Support Ticket page, Question selected in Type dropdown menu" src="https://user-images.githubusercontent.com/1263370/113014673-b2630600-9174-11eb-88ae-ed3cc6b98078.png">
+![Altis Dashboard support link](./assets/altis-support-button.png)
+
+From this screen you'll be able to view your open issues, all previous issues, and you can also query open and previous issues if you're looking for something specific. 
+
+To open a new support ticket, simply click the blue 'Create New Ticket' button.
+
+## Create Support Ticket
+
+A quick overview of the of the screen: 
+
+**Application**
+
+Select the or all Application(s)/Environment(s) to which the support ticket concerns.
+
+**Type**
+
+The type of support ticket, of which there are currently 4. Question, Incident, Problem, and Task. We'll go into more detail about which these categories below, but most tickets will generally fall into one of these four. 
```

Comment:
Perhaps it's a regional thing, but "of which" sounds really formal to me. I think this could be more direct.

```suggestion
The type of support ticket. Currently there are four ticket types: Question, Incident, Problem, and Task. We'll go into more detail about which these categories below, but most tickets will generally fall into one of these four. 
```